### PR TITLE
IBM20180430

### PR DESCRIPTION
**Pull request by ScottMooreIBM**
Added CVE-2018-1389, CVE-2018-1430.

### CODE REVIEW

File: 2018/1xxx/CVE-2018-1389.json [hunk 1]
```diff
@@ -1,18 +1,121 @@
...
+         {
+            "name" : "http://www.ibm.com/support/docview.wss?uid=swg22013531",
+            "url" : "http://www.ibm.com/support/docview.wss?uid=swg22013531",
+            "title" : "IBM Security Bulletin 2013531 (API Connect)",
+            "refsource" : "CONFIRM"
+         },
+         {
+            "name" : "ibm-api-cve20181389-info-mod(138213)",
+            "refsource" : "XF",
+            "url" : "https://exchange.xforce.ibmcloud.com/vulnerabilities/138213",
```
Comment (cve-team) [inline 2018/1xxx/CVE-2018-1389.json]: Trying to retrieve the X-Force document returns "No results can be retrieved by using the search term." Is this just a transient problem?

George

File: 2018/1xxx/CVE-2018-1430.json [hunk 2]
```diff
@@ -1,18 +1,124 @@
...
+            "name" : "http://www.ibm.com/support/docview.wss?uid=swg22013058",
+            "title" : "IBM Security Bulletin 2013058 (API Connect)",
+            "url" : "http://www.ibm.com/support/docview.wss?uid=swg22013058",
+            "refsource" : "CONFIRM"
+         },
+         {
+            "title" : "X-Force Vulnerability Report",
+            "name" : "ibm-api-cve20181430-xss(139226)",
+            "refsource" : "XF",
+            "url" : "https://exchange.xforce.ibmcloud.com/vulnerabilities/139226"
```
Comment (cve-team) [inline 2018/1xxx/CVE-2018-1430.json]: Same issue as with the other CVE entry... transient issue?

George

Comment (ScottMooreIBM) [reply 2018/1xxx/CVE-2018-1430.json]: Looks like some database replication problems, I'm looking into it.

Comment (ScottMooreIBM) [reply 2018/1xxx/CVE-2018-1430.json]: The links are live now.

Comment (cve-team) [reply 2018/1xxx/CVE-2018-1430.json]: Thanks. I'll process this PR shortly.

George

**Pull request merged**